gno.land/pkg/gnoweb/markdown/ext_imgfilter.go

```diff
@@ -0,0 +1,55 @@
+package markdown
+
+import (
+	"github.com/yuin/goldmark"
+	"github.com/yuin/goldmark/ast"
+	"github.com/yuin/goldmark/parser"
+	"github.com/yuin/goldmark/text"
+	"github.com/yuin/goldmark/util"
+)
+
+// ImageFilterFunc filter the element if returning `true`
+type ImageFilterFunc func(uri string) bool
```

Comment:
This is counterintuitive to me.

Two ideas:

- `ImageTransformFunc` - `func(uri string) string` - some wording to change, but more explicit, I think this is better longer term
- `ImageValidatorFunc`  - `func(uri string) (ok bool)` - like this one, but works the opposite way (true = valid, kept; false = invalid, removed)

Reply:
Hmmm, I'm not sure about `ImageTransformFunc`. It makes me feel that this callback would somehow transform the URI, but that's not the case; it only filters it out. I will go for the validator function instead.

Reply:
5ce7776b21035811a80f8d1c06cde1cae894db87